tests/unit/utils/test_lines.py

```diff
@@ -177,3 +177,18 @@ def test_new_line_added_short_text_before_list():
 - Rain
 - Snow"""
     assert lines.wrap(input, width=60) == expected
+
+
+def test_new_line_preserved_short_text_before_list_without_colon():
+    input = """Today's forecast will have different weather.
+
+- A mix of hail and snow, followed by rain clouds, then finally clear sky
+- Rain
+- Snow"""
+    expected = """Today's forecast will have different weather.
+
+- A mix of hail and snow, followed by rain clouds, then
+  finally clear sky
+- Rain
+- Snow"""
```

Comment:
I would text something like the sample test in my previous comment: Multi-paragraph list items; non-list text following list; sublists.

Reply:
I had to apply another fix to get your sample test to pass . I'll open a separate PR

Reply:
The fix is only a 1 line change so I included it in this PR in edef256, with the additional sample test.

Note in the sample, I had to change
```
- Integer rhoncus varius 
  Ante, ac tempus augue finibus sit amet. Integer ac fermentum neque, a sodales nibh. Mauris et dictum ipsum. Integer sit amet posuere urna. Nullam cursus molestie posuere. Praesent imperdiet cursus purus, in posuere odio. 
```
to
```
- Integer rhoncus varius. Ante, ac tempus augue
finibus sit amet. Integer ac fermentum neque, a sodales nibh. Mauris et dictum ipsum. Integer sit amet posuere urna. Nullam cursus molestie posuere. Praesent imperdiet cursus purus, in posuere odio. 
```
for it to be treated as the same list item, otherwise it is treated as a separate paragraph.